### PR TITLE
Fix release workflow yaml

### PR DESCRIPTION
**Pull request by doriable**
This fixes the formatting on the release workflow.

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -91,7 +91,8 @@ jobs:
         with:
           app_id: ${{env.APP_ID}}
           private_key: ${{ secrets.TOKEN_EXCHANGE_GH_APP_PRIVATE_KEY }}
-          repositories: [ ${{ github.repository }} ]
+          repositories:
+            - ${{ github.repository }} ]
```
Comment (joshcarp) [inline .github/workflows/release.yaml:95]:
```suggestion
            - ${{ github.repository }} 
```

Comment (oliversun9) [inline .github/workflows/release.yaml:95]: Do we still need the `]`?

Comment (doriable) [reply .github/workflows/release.yaml:95]: No definitely not, moved through this a bit too quickly! ^^"

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -320,7 +327,8 @@ jobs:
         with:
           app_id: ${{secrets.BUF_BUILD_RELEASE_APP_ID}}
           private_key: ${{ secrets.BUF_BUILD_RELEASE_APP_PRIVATE_KEY }}
-          repositories: [ "bufbuild/buf.build" ]
+          repositories:
+            "bufbuild/buf.build"
```
Comment (joshcarp) [inline .github/workflows/release.yaml:331]:
```suggestion
            - "bufbuild/buf.build"
```

**Review by joshcarp — APPROVED** [review]

**Pull request merged**